terraform.tfvars

```diff
@@ -2,6 +2,6 @@ env_code = "main"
 
 vpc_cidr = "10.0.0.0/16"
 
-public_cidr = ["10.0.0.0/24","10.0.1.0/24"]
+public_cidr = ["10.0.0.0/24", "10.0.1.0/24"]
 
-private_cidr = ["10.0.2.0/24","10.0.3.0/24"]
\ No newline at end of file
+private_cidr = ["10.0.2.0/24", "10.0.3.0/24"]
```

Comment:
Each file should end with a newline. Please add a newline to the end of this file